src/components/Result.tsx

```diff
@@ -56,6 +63,13 @@ const Result = () => {
                   >
                     {`${3 - result.question.remainingAttempts}`}
                   </td>
+                  <td className={styles.td}>
```

Comment:
Does this look center-aligned or should we also use the `centered` class, similar to what we do with the attempts? (same some lines above fot the `th`)

Reply:
![Screenshot 2024-04-17 at 09 47 36](https://github.com/christophblessing/quiz-recap/assets/57761061/199fd396-b3d4-4852-a204-e191f7c432e2)

**I'd say it looks better center-aligned as you proposed.** 

![Screenshot 2024-04-17 at 09 47 17](https://github.com/christophblessing/quiz-recap/assets/57761061/e080f93d-e6f1-4da2-b63a-8f48108bb9fe)

Reply:
I agree 👍 Please also update the screenshot in the description.